diff --git a/node_modules/next/dist/server/config.js b/node_modules/next/dist/server/config.js
index f20b573..ffae183 100644
--- a/node_modules/next/dist/server/config.js
+++ b/node_modules/next/dist/server/config.js
@@ -526,8 +526,11 @@ function assignDefaults(dir, userConfig, silent = false) {
         },
         ahooks: {
             transform: {
-                createUpdateEffect: "modularize-import-loader?name=createUpdateEffect&from=named&as=default!ahooks/es/createUpdateEffect",
-                "*": "ahooks/es/{{member}}"
+                createUpdateEffect: 'modularize-import-loader?name=createUpdateEffect&from=named&as=default!ahooks/es/createUpdateEffect',
+                clearCache: 'modularize-import-loader?name=clearCache&from=named&as=default!ahooks/es/useRequest',
+                useResponsive: 'modularize-import-loader?name=useResponsive&from=named&as=default!ahooks/es/useResponsive',
+                configResponsive: 'modularize-import-loader?name=configResponsive&from=named&as=default!ahooks/es/useResponsive',
+                '*': 'ahooks/lib/{{member}}',
             }
         },
         "@ant-design/icons": {
